site/assignments/starbucks/starbucks.rst

```diff
@@ -136,12 +136,15 @@ Part 5 --- Counting Starbucks
 =============================
 
 In order to calculate the density of Starbucks, the number of Starbucks within the specified area needs to be known.
-With the data available, the way to do this is to check each Starbucks' latitude & longitude and check if it falls
+With the data available, the way to do this is to check each Starbucks' latitude & longitude and see if it falls
 within the specified "rectangle". In other words, check if the Starbucks' latitude falls between the "rectangle's"
 latitudes and if the longitude falls between the "rectangle's" longitudes. See the below image for an example.
 
 .. image:: a1-LatLongSquareCount.png
 
+For the purpose of this assignment, only count Starbucks if they are strictly between the specified latitudes and
+longitudes; don't count them if they fall on any of the latitude/longitude lines. 
```

Comment:
I'm not sure if first years know the "strictly" jargon. May want to be verbose and say something like "that means < and not <="

Reply:
I think by this time I'll have used it, but I'll add something to make it clearer.